package.json

```diff
@@ -36,64 +35,53 @@
     "launchdarkly-js-client-sdk": "^2.24.2",
     "lodash": "^4.17.21",
     "lodash.omit": "^4.5.0",
-    "pdfjs-dist": "3.4.120",
+    "pdfjs-dist": "3.9.179",
     "pinia": "^2.0.35",
     "pinia-class": "^0.0.3",
     "provinces": "^1.11.0",
     "qrcode.vue": "1.7.0",
     "regenerator-runtime": "^0.13.11",
     "register-service-worker": "^1.7.2",
     "sbc-common-components": "3.0.8",
-    "vue": "2.7.10",
+    "vue": "2.7.14",
     "vue-affix": "^0.5.2",
     "vue-hotjar": "^1.4.0",
     "vue-router": "^3.6.5",
     "vue2-filters": "^0.14.0",
-    "vuelidate": "^0.7.7",
+    "vuelidate": "0.6.2",
     "vuetify": "^2.6.15",
     "vuex": "^3.6.2"
   },
   "devDependencies": {
-    "@babel/plugin-proposal-private-methods": "^7.18.6",
     "@pinia/testing": "^0.0.16",
-    "@types/jest": "^27.5.2",
     "@types/lodash": "^4.14.194",
-    "@types/pdfjs-dist": "^2.10.378",
     "@types/sinon": "^10.0.14",
     "@types/vue-the-mask": "^0.11.1",
     "@types/vuelidate": "^0.7.16",
     "@typescript-eslint/eslint-plugin": "^5.59.2",
     "@typescript-eslint/parser": "^5.59.2",
+    "@vitejs/plugin-vue2": "^2.2.0",
     "@volar-plugins/vetur": "latest",
-    "@vue/cli-plugin-babel": "^5.0.8",
-    "@vue/cli-plugin-eslint": "^5.0.8",
-    "@vue/cli-plugin-typescript": "^5.0.8",
-    "@vue/cli-plugin-unit-jest": "^5.0.8",
-    "@vue/cli-service": "^5.0.8",
     "@vue/eslint-config-standard": "^4.0.0",
     "@vue/eslint-config-typescript": "^9.1.0",
     "@vue/test-utils": "^1.3.5",
-    "@vue/vue2-jest": "^27.0.0",
     "eslint": "~7.32.0",
     "eslint-plugin-vue": "^8.7.1",
     "eslint-plugin-vuetify": "^1.1.0",
     "flush-promises": "^1.0.2",
+    "happy-dom": "^10.5.2",
     "sass": "^1.55.0",
-    "sass-loader": "^10.4.1",
     "sinon": "^14.0.2",
-    "ts-jest": "^27.1.5",
     "typescript": "4.5.5",
+    "vite": "4.3.9",
+    "vite-plugin-environment": "^1.1.3",
+    "vitest": "0.33.0",
     "vue-class-component": "^7.2.6",
-    "vue-cli-plugin-vuetify": "^2.5.8",
     "vue-debounce-decorator": "^1.0.1",
     "vue-plugin-helper-decorator": "^0.0.12",
     "vue-property-decorator": "^9.1.2",
```

Comment:
We don't need `vuelidate-property-decorators` here? Not even for the shared BaseAddress component?

Reply:
I agree but weirdly enough, it's working perfectly. I'm going to change that actually to stay consistent between all the UIs.

Reply:
I just noticed, I didn't update the BaseAddress component. It's still using the one from sbc-common-components. I'll fix that.